docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+```
+
+#### Pull the new branch and check it out on Databricks Repos
+
+Navigate to your Databricks Repo in the Databricks Workspace UI. To pull the changes from your remote repository, look for the `Pull` button in the upper-right corner of the Repo page. Click it to sync the latest changes from the remote repository into your Databricks Repo.
+
+Next, check the available branches. Click on the "Branches" button in the upper-left corner of the Repo page. You should see the newly pulled branch `feat/modified-classifier` in the list. Check out the `feat/modified-classifier` branch by clicking on it in the drop-down list. Your active branch is now feat/modified-classifier.
+
+#### Re-run your project after your changes
+
+Return to your Databricks notebook. Re-run the cells you used to run your project previously. The project will now run again, giving output similar to the following:
+
+```bash
+...
+[08/09/22 11:23:30] INFO     Model has accuracy of 1.000 on test data.                                        nodes.py:74
+                    INFO     Saving data to 'metrics' (MetricsDataSet)...                             data_catalog.py:382
+                    INFO     Completed 3 out of 3 tasks                                           sequential_runner.py:85
+                    INFO     Pipeline execution completed successfully.                                      runner.py:89
+```
+
+You can see that your model's accuracy has changed now that you are using a different classifier to produce the result. As it seems that the accuracy of your classifier on test data has improved, you decide to merge these changes to the `main` branch of your project.
+
+```{note}
+If your cluster terminates, you must re-run the cell containing the `%pip ...` magic to re-install your project's dependencies. If not, repeating this step is only necessary if your project's requirements change.
+```
+
+#### Merge the modified code with the main branch
+
+To merge changes from the `feat/modified-classifier` branch into the main branch of your repository, use Git in your local environment. First, check out the main branch:
+
+```bash
+git checkout main
+```
+
+Now, merge the `feat/modified-classifier` branch:
+
+```bash
+git merge feat/modified-classifier
+```
+
+Now that the main branch of your local Git repository contains the changes you made, you should sync the main branch of your repository with your remote Git provider. At the command line in your local development environment:
+
+```bash
+git push origin
+```
+
+To sync the main branch to your Databricks Repo, navigate to your Databricks Repo in the Databricks Workspace UI and use the `Pull` button as you did previously.
+
+```{note}
+In a project with collaborators, the best practice for merging your work with the main branch is to open a pull request (PR) using your Git provider's interface. PRs enable your collaborators to review your work before merging it.
+```
+
```

Comment:
I think it would be useful to mention what steps users would need to repeat when cluster/processes have been restarted. E.g. the data upload to DBFS is ephemeral right?

Reply:
DBFS is not ephemeral, the only steps that have to be repeated are in the notebook. I will add these.